exchanges/gateio/gateio_types.go

```diff
@@ -435,3 +440,40 @@ type WebsocketTrade struct {
 	Amount float64 `json:"amount,string"`
 	Type   string  `json:"type"`
 }
+
+// WebSocketBalance holds a slice of WebsocketBalanceCurrency
+type WebSocketBalance struct {
```

Comment:
Please fix the WebSocket -> Websocket consistency. We tend to use the latter accross the codebase already.